django_x509/static/django-x509/js/switcher.js

```diff
@@ -0,0 +1,53 @@
+(function($){
+  // function for operation_type switcher
+  var filter_div = function(operation_type){
+    //for each div in the form
+    $(".form-row").each(function(i, obj){
+      $(this).show()
+      obj_class = $(this).attr("class")
+      //if default
+      if (operation_type == '-----'){
+        if( obj_class != "form-row field-operation_type" && obj_class != "form-row field-name"){
+          $(this).hide()
+        }
+        }
+      //if import
+      if (operation_type == 'import'){
+        if(obj_class != "form-row field-certificate"
+        && obj_class != "form-row field-private_key"
+        && obj_class != "form-row field-operation_type"){
+          $(this).hide()
+        }
+      }
+      //if new
+      if (operation_type == 'new'){
+        if(obj_class == "form-row field-certificate"
+        || obj_class == "form-row field-private_key"
+        && obj_class != "form-row field-operation_type"){
+          $(this).hide()
+        }
+      }
+    })
+  }
+  $(document).ready(function(){
+    //dont hide if edit cert
+    var path = window.location.pathname
+    if(path.indexOf("change") == -1){
+      //if field is set
+      if ($(".field operation_type" != "-----")){
+       filter_div($(".field-operation_type").find(":selected").text())
+      }else{
+        //filter default if field is not set
+        filter_div('-----')
+      }
+      $(".field-operation_type").on('change',function(){
+        switcher = $(".field-operation_type").find(":selected").text()
+        filter_div(switcher)
+      })
+    } else{
```

Comment:
leave a space before the opening curly brackets, eg: `} else {`, please apply to all blocks of code in the JS file